floodwatch/src/js/components/FilterParent.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import $ from 'jquery';
+import {GraphParent} from './GraphParent';
+import _ from 'lodash';
+import d3 from 'd3'
+
+type Props = {
+  side: string;
+  currentTopic: string;
+  updateMouseOver: Function;
+}
+
+export class FilterParent extends Component {
+  props: Props;
+
+  constructor(props: Props) {
+    super(props);
+  }
+
+  stackData(data) {
+    let topics = Object.keys(data);
+    let intermediate = topics.map(function(key) {
```

Comment:
This is a great example of where typing can help you - Flow is smart enough to understand that the return value of this `map` call will be `Array<Array<{x: number, y: number, name: string}>>` and then if the input type to `dataStackLayout` is exactly that, it helps catch errors (if any - this code looks good to me).